### PR TITLE
updated docs for 6.8 (regressions removed)

### PR DESCRIPTION
**Pull request by logstashmachine**

### CODE REVIEW

**Review by karenzone — CHANGES_REQUESTED** [review]
DO NOT MERGE. Contains regressions to docs that will break doc build if merged.  Also contains ECS info not applicable for 6.8.

**Comment by yaauie** [comment]
ECS info _is_ applicable to these plugins, which _individually_ can be configured to run in ECS mode even on Logstash 6.x.

The difference is that Logstash 6.x (and in fact Logstash < 7.9) provides no way to configure the ECS-compatibility modes for a whole pipeline at once with `pipeline.ecs_compatibility`. Since the setting is absent, each plugin will default to `ecs_compatibility => disabled` but the user can override this on a per-plugin-instance basis.

**Comment by karenzone** [comment]
Hmmmm... IIRC, that content is "baked in" the `main` docs for all plugins. I'll dig deeper.

File: docs/plugins/outputs/elasticsearch.asciidoc
Comment (karenzone) [inline docs/plugins/outputs/elasticsearch.asciidoc:317]:
```suggestion
- for elasticsearch clusters 7.x and above: the value of '_doc' will be used;
- for elasticsearch clusters 6.x: the value of 'doc' will be used;
```

**Comment by karenzone** [comment]
**UPDATE**
I fixed the usual 6.8 suspects (input-file and output-elasticsearch), but pulling in the ECS content to the 6.8 branch has broken additional links because that attribute (apparently) wasn't implemented for the 6.8 branch: 
```
16:23:48 INFO:build_docs:Bad cross-document links:
16:23:48 INFO:build_docs:  /tmp/docsbuild/target_repo/html/en/logstash/6.8/plugins-codecs-avro.html contains broken links to:
16:23:48 INFO:build_docs:   - en/ecs/{ecs_version}
16:23:48 INFO:build_docs:  /tmp/docsbuild/target_repo/html/en/logstash/6.8/plugins-codecs-csv.html contains broken links to:
16:23:48 INFO:build_docs:   - en/ecs/{ecs_version}
16:23:48 INFO:build_docs:  /tmp/docsbuild/target_repo/html/en/logstash/6.8/plugins-inputs-http.html contains broken links to:
16:23:48 INFO:build_docs:   - en/ecs/{ecs_version}
16:23:48 INFO:build_docs:  /tmp/docsbuild/target_repo/html/en/logstash/6.8/plugins-inputs-jms.html contains broken links to:
16:23:48 INFO:build_docs:   - en/ecs/{ecs_version}
```

There are several approaches we could use. I'll figure out the best approach, and fix.

**Comment by karenzone** [comment]
Restesting after merging PR to add attribute to 6.8: 

@elasticmachine run elasticsearch-ci/docs

**Review by karenzone — APPROVED** [review]
Regressions removed and broken cross-doc links resolves. Builds cleanly and renders as expected.

**Pull request merged**
